.dockerignore

```diff
@@ -4,3 +4,12 @@ bench
 test
 images
 lib
+.clinic
+.nyc_output
+coverage
+docs
+docsify
+types
+.husky
+examples
+benchmarks
```

Comment:
Oh, I missed a lot of things 😄 . We could also use a negative approach.

For example:
```
*
!index.js
```

will ignore everything bu `index.js`, like `.gitignore`.